jsonnet/components/prometheus-operator-user-workload.libsonnet

```diff
@@ -113,6 +111,22 @@ function(params)
         },
       },
     },
+    prometheusRule+: {
```

Comment:
Should we consider reusing the same alerts defined in PM namespace for prometheus(by modifying [prometheusOperatorSelector](https://github.com/prometheus-operator/kube-prometheus/blob/main/jsonnet/kube-prometheus/components/prometheus-operator.libsonnet#L33) config) instead of redefining the alerts again in UWM namespace?

i.e.
```
 prometheusOperatorSelector: 'job="prometheus-operator",namespace~="openshift-monitoring|openshift-user-workload-monitoring"'
```

Reply:
It shall be done just by adding `prometheusOperatorSelector` config value in https://github.com/openshift/cluster-monitoring-operator/blob/master/jsonnet/main.jsonnet#L273.

```
prometheusOperatorSelector: 'job="prometheus-operator", namespace=~"%(namespace)s|%(namespaceUserWorkload)s"' % $.values.common,
```

Reply:
yes we should rather tweak `prometheusOperatorSelector` 
/hold

Reply:
Closing out this PR and will raise a fresh one with the new approach.